src/StratumClient/StratumClient.fs

```diff
@@ -0,0 +1,42 @@
+namespace TcpEcho
+
+open System.IO
+open System.Text
+open Newtonsoft.Json
+
+type BlockchainTransactionGet = {
+    [<JsonProperty(PropertyName = "id")>]
+    mutable Id: int
+    [<JsonProperty(PropertyName = "result")>]
+    mutable Result: string
```

Comment:
are you sure you need the elements of the record to be mutable? I think there is no need

Reply:
The C# class is public and has getters and setters. I don't know whether the setters are actually intended to be used anywhere, but I was trying to replicate the C# code faithfully.